packages/terra-theme-properties/src/scss/semantic-names.scss

```diff
@@ -0,0 +1,214 @@
+// terra-redwood-theme
+// Version 0.20.12
+// Semantic Names for Terra Custom Properties
+// ENGINEERING NOTE:
+// Only reference variables from this file.
+// Terra components should not reference any variable from
+// files contained in the `protected` directory.
+// -------------------------------------------------
+// Maintained by Oracle Health Design, UX Foundations Team
+// Do not edit these values directly
+// -------------------------------------------------
+
+// =======================================
+// Default Values
+// =======================================
+.redwood-theme {
+  // Focus ring styles
+  --default-focus-outline-style: var(--rds-focus-outline-style);
+  --default-focus-outline-width: var(--rds-focus-outline-width);
+  --default-focus-outline-offset-inside: var(--rds-focus-offset-inside);
+  --default-focus-outline-offset-outside: var(--rds-focus-offset-outside);
+  --default-focus-outline-color-inverse: var(--rds-focus-inverse-outline-color);
+  --default-focus-outline-color-normal: var(--rds-focus-normal-outline-color);
+}
+
+// =======================================
+// terra-badge
+// =======================================
+.redwood-theme {
+  // Color -> Neutral
+  --badge-neutral-background-color: var(--ui-light-color-surface-neutral-subtle);
+  --badge-neutral-text-color: var(--ui-light-color-text-icon-secondary);
+  --badge-neutral-icon-color: var(--ui-light-color-text-icon-secondary);
+
+  // Color -> Positive
+  --badge-positive-background-color: var(--ui-light-color-surface-success-subtle);
+  --badge-positive-text-color: var(--ui-light-color-text-icon-success);
+  --badge-positive-icon-color: var(--ui-light-color-text-icon-success);
+
+  // Color -> Negative
+  --badge-negative-background-color: var(--ui-light-color-surface-danger-subtle);
+  --badge-negative-text-color: var(--ui-light-color-text-icon-danger);
+  --badge-negative-icon-color: var(--ui-light-color-text-icon-danger);
+
+  // Color -> Warning
+  --badge-warning-background-color: var(--ui-light-color-surface-warning-subtle);
+  --badge-warning-text-color: var(--ui-light-color-text-icon-warning);
+  --badge-warning-icon-color: var(--ui-light-color-text-icon-warning);
+
+  // Color -> Information
+  --badge-information-background-color: var(--ui-light-color-surface-info-subtle);
+  --badge-information-text-color: var(--ui-light-color-text-icon-info);
+  --badge-information-icon-color: var(--ui-light-color-text-icon-info);
+
+  // Size -> Tiny
+  --badge-tiny-font-size: var(--rds-typography-font-size-xs);
+  --badge-tiny-icon-size: var(--rds-icon-size-2xs);
+  --badge-tiny-radius: var(--rds-border-radius-sm);
+  --badge-tiny-left-padding: var(--rds-spacing-3);
```

Comment:
Shouldn't padding go before right/left, so that the naming is following: --badge-tiny-padding-left for all padding variables?

Reply:
i'll bring these points up with the UX team when we meet with them.  We are not controlling these names at the moment, but I get your points.